Makefile

```diff
@@ -242,7 +242,7 @@ lint_quick:
 
 # Globally disable the following pylint messages:
 PYLINT_DISABLE := \
-	import-error redefined-outer-name unused-argument \
+	consider-using-with import-error redefined-outer-name unused-argument \
```

Comment:
hmm, this change shouldn't be part of the hwl

Reply:
Hi Avery, I put this in because I was having trouble with pylint in pushing my code. It is a temporary solution suggested by Mitchell, but I am looking into a more permanent solution. Other than that, I have applied all your suggestions. Thank you for the feedback! If there is anything else I should know or do, please let me know.